docs/customizations.md

```diff
@@ -47,6 +47,28 @@ Each value must be prefixed by `EXPO_PUBLIC_`. See list of available customizati
 
 `EXPO_PUBLIC_SPLASH_IMAGE`: custom splash screen image
 
+`EXPO_PUBLIC_${ANDROID_TV | APPLE_TV}_*`: various images for Android TV and tvOS apps such as the banner for android TV
+or background images for Apple TV.
+
+> [!WARNING]
+> You need to supply images of expected sizes for tvOS or the build fails automatically:
+> 
+> EXPO_PUBLIC_APPLE_TV_ICON: 1280x760
+> 
+> EXPO_PUBLIC_APPLE_TV_ICON_SMALL: 400x240
+> 
+> EXPO_PUBLIC_APPLE_TV_ICON_SMALL_2X: 800x480
+> 
+> EXPO_PUBLIC_APPLE_TV_TOP_SHELF: 1920x720
+> 
+> EXPO_PUBLIC_APPLE_TV_TOP_SHELF_2X: 3840x1440
+> 
+> EXPO_PUBLIC_APPLE_TV_TOP_SHELF_WIDE: 2320x720
+> 
+> EXPO_PUBLIC_APPLE_TV_TOP_SHELF_WIDE_2X: 4640x1440
+> 
+> More info at https://www.npmjs.com/package/@react-native-tvos/config-tv
+
```

Comment:
Good addition! For clarity, what do you think about including a `_WxH` suffix in file names? E.g. `./OwnTube.tv/assets/appleTV/iconSmall2x_800x480.png`? We could use this convention for all assets, or only for assets where it's a breaking bug to submit wrong dimensions.